api/src/app/modules/submission/submission.dto.ts

```diff
@@ -19,6 +19,12 @@ export enum SpatialObjectCodeEnum {
   WTRA = 'WTRA'
 }
 
+export enum SpatialObjectTableEnum {
+  CUT_BLOCK = 'cut_block',
+  ROAD_SECTION = 'road_section',
+  WTRA = 'retention_area'
+}
```

Comment:
The dto has another similar enum `SpatialObjectCodeEnum`.
I might avoid having similar enum class (have almost the same values) to prevent confusion.
Could just have a little util function to return table name, or just the same as before, the util function to use `switch` to return composed sql query based on `spatialObjectCode`.

Or, if you still prefer the enum class you created, maybe add a comment on top to make it clear it is not the same purpose of the previous one, just don't mix it up for other developers (who might not be careful)...

Reply:
this is removed.